core/text/code_formatter.talon-list

```diff
@@ -4,6 +4,8 @@ all cap: ALL_CAPS
 all down: ALL_LOWERCASE
 camel: PRIVATE_CAMEL_CASE
 dotted: DOT_SEPARATED
+listed: COMMA_SEPARATED
```

Comment:
We already have a formatter "list" in the reformat Talon list. If you want to be able to insert we should just move that to code formatters instead.

Reply:
I can't find where that reformatter list gets used in the docs, nor in any cheatsheets - How does it get used? Why would we want to move 'list' as a verb rather than adding a new transitive verb 'listed'?

Reply:
Everything that uses the formatters capture. eg
https://github.com/talonhub/community/blob/25cf36af08984a74acf4b4127a521621de5bfafa/core/text/text.talon#L16

Because we already have a spoken form for this and we don't want two different spoken forms for the same thing. Either leave it as a reformatter or move it to code formatters.